docs/user_guide.md

```diff
@@ -119,4 +145,23 @@ Push the image:
 $ docker push 10.117.169.182/demo/ubuntu:14.04  
 ```  
 
-**Note: Replace "10.117.169.182" with the IP address or domain name of your Harbor node.**
\ No newline at end of file
+**Note: Replace "10.117.169.182" with the IP address or domain name of your Harbor node.**
+
+##Deleting repositories
+
+Repositories deletion runs in two steps.  
+First, delete repositories in Harbor's UI. This is soft deletion. You can delete the entire repository or just a tag of it.  
+
+![browse project](img/new_delete_repository.png)
+
+**Note: If both tag A and tag B reference the same image, after deleting tag A, B will also disappear.**  
+
+Second, delete the real data using registry's garbage colliection(GC). 
+

```

Comment:
1. add some explanation of this command, and if harbor should be stopped or assume no push operation is allowed. 

2. state on which machine this command should be run